.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: chimedb-ci-build
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+
+jobs:
+
+  lint-code:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install black
+      run: pip install black
+
+    - name: Check code with black
+      run: |
+        black --check .
+
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install pytest
+      run: |
+        pip install pytest
+
+    - name: Install chimedb
+      run: |
+        pip install .
```

Comment:
Do you need to install with the `[tests]` extension? I think that causes problems with the new pip. =(

Reply:
It does seem to work